src/PlayerStrategies/PlayerStrategies.cpp

```diff
@@ -2,20 +2,126 @@
+    return "Moderate Computer";
+}
+
+// Utility Methods for Driver
+
+bool validateChangeStrategy() {
+    std::string answer;
+    bool invalidAnswer = true;
+
+    std::cout << "Would you like to change or set any player strategies (y/n)? ";
+    do {
+        std::cin >> answer;
+        if (answer == "y" || answer == "n") {
+            invalidAnswer = false;
+        }
+        else {
+            std::cout << "Please enter a 'y' or 'n': ";
+        }
+    } while (invalidAnswer);
+
+    return (answer == "y");
+}
+
+void changePlayerStrategy(std::vector<Player*> players) {
+    bool continueChangingStrategy = true;
+
+    while (continueChangingStrategy) {
+        // display all players and their current strategies
+        std::cout << "Please select a player to change their strategy (1 - " << players.size() << "): " << std::endl;
+        for (int i = 1; i-1 < players.size(); i++) {
+            std::cout << i << ") " << players.at(i-1)->getName() << ": " << players.at(i-1)->getPlayerStrategies()->displayCurrentStrategy() << std::endl;
+        }
+
+        int playerSelection;
+
+        // allow user to select a player to change their strategy
+        std::cout << std::endl;
+        std::cin >> playerSelection;
+        while (playerSelection < 1 || playerSelection > players.size()) {
+            std::cout << "Please enter a number from (1 - " << players.size() << "): ";
+            std::cin >> playerSelection;
+        }
+
+        // display choices
+        std::cout << std::endl << "Please selection that player's new stratedgy: " << std::endl;
+        std::cout << "1) Human Player" << std::endl << "2) Greedy Computer" << std::endl << "3) Moderate Computer" << std::endl;
+
+        int strategySelection;
+
+        // allow user to select a new strategy
+        std::cout << std::endl;
+        std:: cin >> strategySelection;
+        while (strategySelection < 1 || strategySelection > 3) {
+            std::cout << "Please enter a number from (1 - 3): ";
+            std::cin >> strategySelection;
+        }
+
+        // change player strategy
+        switch(strategySelection) {
+            case (1):
+                players.at(playerSelection-1)->setPlayerStrategy(new HumanPlayer());
+                std::cout << players.at(playerSelection-1)->getName() << " is now a Human Player." << std::endl;
+                break;
+            case (2):
+                players.at(playerSelection-1)->setPlayerStrategy(new GreedyComputer());
+                std::cout << players.at(playerSelection-1)->getName() << " is now a Greedy Computer." << std::endl;
+                break;
+            case (3):
+                players.at(playerSelection-1)->setPlayerStrategy(new ModerateComputer());
+                std::cout << players.at(playerSelection-1)->getName() << " is now a Moderate Computer." << std::endl;
+                break;
+        }
 
+        std::cout << "Would you like to change the strategy of another player? ";
```

Comment:
`std::cout << "Would you to change the strategy of another player?\n ";`

make this sentence and next sentence two line display

Reply:
I second this, you can probably remove this line entirely. For reference, when we play this is what we see:

```
Would you like to change the strategy of another player? Would you like to change or set any player strategies (y/n)?
```

Reply:
Done.